Scriptblock/sbBrowser.ps1

```diff
@@ -0,0 +1,51 @@
+﻿
```

Comment:
Just doing Chrome for now ... it's the one I for which I enhanced the Plaso list